script.js

```diff
@@ -0,0 +1,100 @@
+const hamburger = document.getElementById('menuBtn');
+const quitBtn = document.querySelector('.btnClose');
+const menuPopup = document.getElementById('menu-bar');
+const menuItems = document.querySelectorAll('.menu-item');
+
+menuPopup.classList.add('hidden');
+
+function closePopup() {
+  hamburger.classList.toggle('hidden');
+  quitBtn.classList.toggle('hidden');
+  menuItems.forEach((item) => { menuPopup.remove(item); });
+  menuPopup.classList.toggle('hidden');
+}
+
+function menuDisplay() {
+  hamburger.classList.toggle('hidden');
+  quitBtn.classList.toggle('hidden');
+  menuPopup.classList.add('menuPopup');
+  menuPopup.classList.remove('hidden');
+
+  menuPopup.appendChild(quitBtn);
+  menuItems.forEach((item) => {
+    menuPopup.appendChild(item);
+    item.addEventListener('click', closePopup);
+  });
+  quitBtn.addEventListener('click', closePopup);
+}
+
+hamburger.addEventListener('click', menuDisplay);
+
+// speakers cards //
+
+const cards = document.querySelector('.container-article');
+
+const speakers = [
+  {
+    id: 'card1',
```

Comment:
- [ ] Please make sure to use the same name for your object keys, as here you are using `id` but on the other ones you are using `cardid`. Same thing with the `soloist` key here and the `speaker` key on the rest of the objects.